File: app/src/main/java/com/kota/Telnet/TelnetArticlePage.java
package com.kota.Telnet;

import com.kota.Telnet.Model.TelnetRow;
import java.util.Vector;

public class TelnetArticlePage {
    private Vector<TelnetRow> rows = new Vector<>();

    public void addRow(TelnetRow row) {
        this.rows.add(row.clone());
    }

    public int getRowCount() {
        return this.rows.size();
    }

    public TelnetRow getRow(int index) {
        return this.rows.get(index);
    }

    public void clear() {
        this.rows.clear();
    }
}

File: app/src/main/java/com/kota/telnet/TelnetArticlePage.kt
package com.kota.telnet

import com.kota.telnet.model.TelnetRow
import java.util.Vector

class TelnetArticlePage {
    val rows = Vector<TelnetRow>()

    fun addRow(row: TelnetRow) {
        this.rows.add(row.clone())
    }

    val rowCount: Int
        get() = this.rows.size

    fun getRow(index: Int): TelnetRow {
        return this.rows[index]
    }

    fun clear() {
        this.rows.clear()
    }
}
